src/AVFoundation/AVCaptureVideoPreviewLayer.cs

```diff
@@ -0,0 +1,37 @@
+#if !WATCH && !TVOS
+
+using System;
+
+using XamCore.Foundation;
+using XamCore.CoreFoundation;
+using XamCore.ObjCRuntime;
+using XamCore.AudioToolbox;
+
+namespace XamCore.AVFoundation {
+	public partial class AVCaptureVideoPreviewLayer {
+
+		[iOS (8,0), Mac (10,2)]
```

Comment:
^ wrong place for the attribute, `WithConnection` should be working (back to 4.0). It's only `WithNoConnection` which is restricted to newer OS